compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/insert_dma_bd_chain_disable_interleave.mlir

```diff
@@ -0,0 +1,156 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(iree-amdaie-insert-dma-bd-chain{enable-interleave=false})" --split-input-file --verify-diagnostics %s | FileCheck %s
+
+// There is only a single BD chain anyway.
+// Same results no matter `enable-interleave` is true or false.
+// CHECK-LABEL: @single_bd_chain
+// CHECK-COUNT-1: amdaie.npu.dma_wait
+// CHECK-NOT: amdaie.npu.dma_wait
```

Comment:
Nit: could you align these checks in the following for readability as we do in most of the repo:
```suggestion
// CHECK-LABEL:    @single_bd_chain
// CHECK-COUNT-1:  amdaie.npu.dma_wait
// CHECK-NOT:      amdaie.npu.dma_wait
```